packages/admin/cypress/integration/super-admin-dashboard/super-admin-dashboard.spec.js

```diff
@@ -0,0 +1,47 @@
+import { login } from '../../utils/login';
+
+describe('super-admin-dashboard', () => {
+  beforeEach(() => {
+    cy.task('db:deleteTestUsers');
+    cy.task('db:addTestUsers');
+    cy.task('wiremock:selectUser', 'superAdmin');
```

Comment:
// we could maybe further abstract this by having one function that runs these three tasks
```js
type User = 'applicant' | 'admin' | 'superAdmin';

const setUpUser = (user: User) => {
  cy.task('db:deleteTestUsers');
  cy.task('db:addTestUsers');
  cy.task('wiremock:selectUser', user);
};
```